app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: inherit;
+}
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $base-font;
+}
```

Comment:
put new line between the styles

Reply:
solved within all ```.css``` files. ✓

Reply:
I just read this, then looked at the next file (footer.scss) and it's not applied, should I trust this? :-)